src/components/atoms/IncompatibleModCard.astro

```diff
@@ -0,0 +1,69 @@
+---
+import { marked } from "marked";
+
+export interface Mod {
+    name: string;
+    ids: string[];
+    icon: string;
+    note?: string;
+    tracking: string;
+    status: string;
+}
+
+export interface Props {
+    mod: Mod;
+}
+
+const mod = Astro.props.mod;
+
+const [statusText, statusIcon] = {
+    "UNKNOWN": ["Status is currently unknown", "fa-question"],
+    "WONT_FIX": ["Will not be fixed", "fa-ban"],
+    "IN_PROGRESS": ["Fix in progress", "fa-spinner"],
+    "BLOCKED": ["Fix is blocked by an external factor", "fa-shield-halved"],
+    "NO_ANSWER": ["The tracking issue is awaiting a response from maintainers", "fa-hourglass-half"],
+    "ON_HOLD": ["Fix is currently on hold", "fa-pause"],
+}[mod.status || "UNKNOWN"]
+---
+<div class="mod">
+    <div class="card">
+        <div class="card-content p-3 mb-3">
+            <div class="media">
+                <div class="media-left">
+                    <figure class="image is-64x64" style="margin-left: 1vw; margin-right: 1vw">
```

Comment:
Haven't looked at the rest of the project, is inline styling standard here ?

Reply:
Yeah, as adding CSS is an absolute pain as it is in entirely in a different repository